app/scripts/lib/hardware-keyring-builder-factory.ts

```diff
@@ -1,13 +1,15 @@
 import type { TrezorBridge } from '@metamask/eth-trezor-keyring';
 import type { LedgerBridge } from '@metamask/eth-ledger-bridge-keyring';
 import { KeyringClass, Json } from '@metamask/utils';
+import { FakeKeyringBridge } from '../../../test/stub/keyring-bridge';
 
 /**
  * A transport bridge between the keyring and the hardware device.
  */
 export type HardwareTransportBridgeClass =
   | (new () => TrezorBridge)
-  | (new () => LedgerBridge);
+  | (new () => LedgerBridge)
+  | (new () => FakeKeyringBridge);
```

Comment:
We should probably create a shared type of `KeyringBridge`, that should be implemented by all bridges: this way we wouldn't need to list every single bridge we might want to use here